UnitsNet.WindowsRuntimeComponent/UnitsNet.WindowsRuntimeComponent.csproj

```diff
@@ -11,25 +11,22 @@
     <AssemblyName>UnitsNet</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.16299.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.10586.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.18362.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.18362.0</TargetPlatformMinVersion>
```

Comment:
Is there a reason to bump up the MinVersion?

Reply:
I think Visual Studio makes this change automatically. But yes, best to revert this.